### PR TITLE
Passwords updated at label

### PR DESCRIPTION
**Pull request by zarkomilosevic**

### CODE REVIEW

File: app/models/password.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 class Password < ApplicationRecord
+
+  include ActiveModel::Dirty
```
Comment (netoff) [inline app/models/password.rb]: Mislim da ovo ne treba da se ukljucuje posebno, da u svakom modelu vec postoje `_changed?`, tj da je `Dirty` već inkludovan. Jel možeš da probaš bez ovoga?

File: app/models/password.rb [hunk 2]
```diff
@@ -16,4 +22,13 @@ def favicon_URI
   def favicon_DataURI
     favicon.try(:data)
   end
+
+  def is_updated?
```
Comment (netoff) [inline app/models/password.rb]: Jel ovo `is_updated?` sliži nečemu, tj. jel se koristi pošto nisam mogao da nađem? Ako se ne koristi onda bi ga trebalo skloniti...

File: app/views/passwords/_password.html.erb [hunk 3]
```diff
@@ -19,6 +19,10 @@
         <span class="password-block-password-data" id="password-data-username-<%= password.id %>">  <b>Username:</b> <%= password.username %> </span>
         <br />
         <span class="password-data password-block-password-data" id="password-data-password-<%= password.id %>">  <b>Password:</b> <%= password.password %> </span>
+        <br />
+        <% if password.password_last_changed_at.present? %>
```
Comment (netoff) [inline app/views/passwords/_password.html.erb]: Možda bi ovde u dugom slučaju trebalo staviti jedan `else` uslov ako `password_last_changed_at` ne postoji i prikazati informaciju u obliku "Password created at <%= time_ago_in_words(password.password_created_at)%> ".

Ili je druga opcija da se prilikom kreiranja `password_last_changed_at` postavi na istu vrednost kao i `created_at` tako da onda uvek bude pristutno. Ne znam šta bi bilo pravilnije...

File: test/support/wait_for_ajax.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+module WaitForAjax
```
Comment (netoff) [inline test/support/wait_for_ajax.rb]: @zarkomilosevic U štini kapibara treba da bude u stanju sama da sačeka da se svi ajax pozivi završe. Ukoliko to nije slučač mislim da onda ne koristimo Capybara-ine helepre na pravi način. Tj da bi trebalo da uradimo refactor samih testova tako da sve proradi kako treba, be uzpotrebe dodatnih mehanizama koji čekaju na ajax. Sada ču da odobrim ovaj merge, ali ću staviti ovde jedan `TODO` komentar da se kasnije ovo reši na propisan način.
